doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+
+!media lfr/LFR_rzmflx.png
+       style=width:70%
+       id=LFR_rzmflx
+       caption=Comparison of TWODANT spectra of different axial regions
+
+!media lfr/LFR_rzm_ufg_spectra.png
+       style=width:70%
+       id=LFR_rzm_ufg_spectra
+       caption=Comparison of zero-dimensional (0D) UFG and TWODANT spectra of different axial regions
+
+### Second step id=xs2nd
+
+The second step is to condense the UFG cross section to the broad group (BG) cross section using the TWODANT spectrum. Cross sections for the activel fuel region and the other regions are prepared separately and merged at the last step.
+
+For the fuel region, fine mesh-based region-wise cross sections need to be generated as the Griffin transport calculation is fine-mesh based. This can be achieved by setting up an MC2-3 input that employs a 1-dimensional (1D) heterogeneous tranport calculation using the collision probability method (CPM) whose model is shown in [onedxs_model]. Hexagonal ring-wise cross sections are generated for fuel (7 sets) and one set of cross sections is generated for each of clad, duct, and lead for inner and outer regions of duct. The annular fuel and helium hole and gap are smeared and its homogenized material is used to build the model as microscopic cross sections of isotopes comprising them are not affected by such volume homogenization.
+
+!media lfr/onedxs_model.png
+       style=width:40%
+       id=onedxs_model
+       caption=1D R-geometry model for hexagonal ring-wise cross section generation (red: fuel + helium, blue: lead, gray: clad, cyan: duct)
+
+Below shows the `control` block. Specifying `c_geometry_type=cylinder` automatically sets the problem type as a 1D CPM transport problem. `i_number_region` means the number of different cross section regions. The self-shielding condition is calculated per cross section region and thus UFG cross sections are unique per cross section region. `c_externalspectrum_ufg=rzmflx` means to use the TWODANT spectrum in the condensation step after transport calculation. For this, the `rzmflx` file needs to be placed in the same folder where the MC2-3 input exists with the filename of `rzmflx` without extension.
```

Comment:
I am confused by this second step. You are performing 1D transport calculation. Looks like you are not using the UFG group structure for this calculation. Then how many groups in this transport calculation? What is that `ANL9` group structure? You mentioned later the TWODANT spectrum is superimposed to the region-wise spectrum, how is the superimposition is done?

Reply:
I added the following for clarification.

```
For clarification, the TWODANT calculation in the first step is the 1041 group calculation. In this secton step, the 0D UFG calculation is performed for each mixture of non-fuel axial regions and the 1D UFG calculation is performed for the fuel axial region by MC2-3. Then, resulting UFG spectra are superimposed to 1041-group TWODANT spectra for condensation of UFG cross sections to the broad group structure specified in the c_group_structure of the &control block.
```